site/docs/master/README.md

```diff
@@ -1,4 +1,4 @@
-# <img src="img/sonobuoy-logo.png" width="400px" > [![CircleCI](https://circleci.com/gh/vmware-tanzu/sonobuoy.svg?style=svg)](https://circleci.com/gh/vmware-tanzu/sonobuoy)
+# <img src="img/sonobuoy-logo.png" alt="Sonobuoy logo" width="400px" > [![CircleCI](https://circleci.com/gh/vmware-tanzu/sonobuoy.svg?style=svg)](https://circleci.com/gh/vmware-tanzu/sonobuoy)
```

Comment:
So CI is failing because you edited this file directly. In order to keep the github README.md in sync with this document we treat the root README.md as the source of truth and script the updating of this one.

```
scripts/sync_readme.sh modified the git status. If updating the README.md, update the root README.md and run that script.
```

Reply:
Thanks mate, 

I've updated the root README.md and run the sync script.